README.md

```diff
@@ -1,5 +1,4 @@
 [![Build Status](https://travis-ci.org/SuperTux/supertux.svg?branch=master)](https://travis-ci.org/SuperTux/supertux)
-[![AppVeyor Build Satus](https://ci.appveyor.com/api/projects/status/github/SuperTux/supertux?svg=true&branch=master)](https://ci.appveyor.com/project/maths22/supertux-9ml4d/branch/master)
```

Comment:
`maths22` should probably be replaced with `supertux` so that it works again.

@maths22 did you change something related to AppVeyor?